### PR TITLE
Update bylabeltext.mdx

### PR DESCRIPTION
**Pull request by ruhollahh**
Updated the example code for React and Cypress so it works properly.

### CODE REVIEW

File: docs/queries/bylabeltext.mdx [hunk 1]
```diff
@@ -71,14 +71,14 @@ import { render, screen } from '@testing-library/react'
 
 render(<Login />)
 
-const inputNode = screen.getByLabelText('username')
+const inputNode = screen.getByLabelText('Username')
```
Comment (alexkrolick) [inline docs/queries/bylabeltext.mdx:74]: Can you update the examples so they are all capitalized the same way?

`L53 <input aria-label="username" />`

**Review by ruhollahh — COMMENTED** [review]
Yes, absolutely.

**Review by eps1lon — APPROVED** [review]
Thanks!

**Comment by eps1lon** [comment]
@all-contributors add @ruhollahh for docs

**Pull request merged**